examples/reason-act.sh

```diff
@@ -0,0 +1,16 @@
+
+#!/bin/bash
+
+cd `dirname $0`
+cd ..
+
+# get -m model parameter otherwise defer to default
+if [ "$1" == "-m" ]; then
+  MODEL="-m $1"
```

Comment:
Wouldn’t this always evaluate to `-m -m`?

Reply:
err, fixed